templates/workspaces/base/porter.yaml

```diff
@@ -126,6 +126,14 @@ parameters:
     type: string
     default: "GRS"
     description: "The redundancy option for the storage account in the workspace: GRS (Geo-Redundant Storage) or ZRS (Zone-Redundant Storage)."
+  - name: enable_airlock_malware_scanning
+    type: boolean
+    default: false
+    description: "Enable malware scanning on the workspace storage account"
+  - name: airlock_malware_scan_result_topic_name
+    type: string
+    default: ""
+    description: "The name of the topic to publish scan results to"
```

Comment:
Since both those params are internal and not up to user discretion, I think we should hide them in the UI. WDYT?
An example: https://github.com/microsoft/AzureTRE/blob/8812503011f73004960529acff20e76fb77fd164/templates/workspaces/airlock-import-review/template_schema.json#L165